src/containers/Teams/index.tsx

```diff
@@ -87,26 +91,18 @@ const Teams: FC = () => {
   }, [history, teamParam]);
 
   useEffect(() => {
-    const getFilteredMembers = (): TeamMember[] => {
-      const teamLeads: TeamMember[] = [];
-      const otherMembers: TeamMember[] = [];
-      teamMembers.forEach((member) => {
-        const {teams} = member;
-        const matchingTeam = teams.find(({title}) => title.toLowerCase() === teamFilter.toLowerCase());
-        if (matchingTeam) {
-          if (matchingTeam.isLead) {
-            teamLeads.push({...member, isLead: true});
-          } else {
-            otherMembers.push({...member, isLead: false});
-          }
-        }
-      });
-      return teamLeads.concat(otherMembers);
-    };
+    const getFilteredMembers = () =>
+      teamMembers.filter((teamMember) => teamMember.team.toLowerCase() === teamFilter.toLowerCase());
```

Comment:
is the order of `lead` and normal team members not matter anymore?

Reply:
what do you mean?

Reply:
@jamessspanggg On line 98, there's `sortTeamMembers` function. That takes care of it.